### PR TITLE
feat: increase flag polling interval; add flag poller interval config

### PR DESCRIPTION
**Pull request by bgiori**
### Summary

* Set the default flag polling interval to 5 min
* Add polling interval config with min value of 1 min

### Checklist

* [x] Does your PR title have the correct [title format](https://github.com/amplitude/experiment-js-client/blob/main/CONTRIBUTING.md#pr-commit-title-conventions)?
* Does your PR have a breaking change?:   no

### CODE REVIEW

File: packages/experiment-browser/src/experimentClient.ts [hunk 1]
```diff
@@ -116,7 +113,18 @@ export class ExperimentClient implements Client {
         (config?.serverZone?.toLowerCase() === 'eu'
           ? euFlagsServerUrl
           : Defaults.flagsServerUrl),
+      // Force minimum flag config polling interval.
+      flagConfigPollingIntervalMillis:
+        config.flagConfigPollingIntervalMillis < minFlagPollerIntervalMillis
+          ? minFlagPollerIntervalMillis
+          : config.flagConfigPollingIntervalMillis
+          ? config.flagConfigPollingIntervalMillis
+          : Defaults.flagConfigPollingIntervalMillis,
```
Comment (zhukaihan) [inline packages/experiment-browser/src/experimentClient.ts:122]: Maybe add parenthesis? 
Also `A ? A : B` == `A ?? B` if we're just testing for null or undefined.

**Review by zhukaihan — APPROVED** [review]

File: packages/experiment-browser/src/experimentClient.ts (hunk 1 above)
Comment (tyiuhc) [inline packages/experiment-browser/src/experimentClient.ts:122]: simplify for legibility:
```
flagConfigPollingIntervalMillis = config.flagConfigPollingIntervalMillis && config.flagConfigPollingIntervalMillis >= minFlagPollerIntervalMillis
  ? config.flagConfigPollingIntervalMillis
  : minFlagPollerIntervalMillis;
```
